src/Xamarin.Android.Build.Utilities/AndroidVersion.cs

```diff
@@ -4,6 +4,7 @@ namespace Xamarin.Android.Build.Utilities
 {
 	public class AndroidVersion
 	{
+		[Obsolete ("Use AndroidVersions.MaxVersion.ApiLevel", error:true)]
```

Comment:
...which doesn't exist. Oops.

Anyway, it's not used in this project, due to `error:true`. Do we really need to track `MaxApiLevel`?